go.mod

```diff
@@ -171,3 +173,5 @@ replace golang.org/x/crypto => golang.org/x/crypto v0.1.0
 replace github.com/containernetworking/cni => github.com/containernetworking/cni v0.8.1
 
 replace github.com/containernetworking/plugins => github.com/containernetworking/plugins v0.9.1
+
+replace github.com/aws/amazon-vpc-resource-controller-k8s => github.com/haouc/amazon-vpc-resource-controller-k8s v1.2.3
```

Comment:
Check out go.work files, which you can add to .gitignore and not need to edit the module: https://go.dev/doc/tutorial/workspaces

Reply:
I will check the doc. My understanding is the multi modules workspace in this case will keep the change local and this PR will fail in GitHub actions? Which is fine since this `replace` change will be removed after the VPC RC has a new tag for CRD update. But I think I get your point.

Reply:
@haouc lets please make sure PR is merged after tag is released, or other option is cut a RC release on VPC RC and use that tag here. Since it will break master builds.

Reply:
Yeah, @jayanthvn. We can discuss offline. This PR has been changed to target release branch as a possible RC release.